app/Http/Controllers/UserController.php

```diff
@@ -112,6 +113,9 @@ public function profile($slug, $id)
         $realdownload = $user->downloaded + $bondownload;
         $invitedBy = Invite::where('accepted_by', '=', $user->id)->first();
 
+        $requested = TorrentRequest::where('user_id','=',$user->id)->count();
+        $filled = TorrentRequest::where('filled_by','=',$user->id)->where('approved_by','is not','null')->count();
```

Comment:
This query is not valid. Should be:
```
$filled = TorrentRequest::where('filled_by','=',$user->id)->whereNotNull('approved_by')->count();
```

Reply:
It's dirty, but it's not not valid.

Fixing.